modules/common/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/apirequest/ApiRequestEnrichmentIntegrationTest.scala

```diff
@@ -49,30 +49,17 @@ class ApiRequestEnrichmentIntegrationTest extends Specification {
       "name": "api_request_enrichment_config",
       "enabled": true,
       "parameters": {
-        "inputs": [
-          {
-            "key": "user",
-            "pojo": {
-              "field": "user_id"
-            }
-          },
-          {
-            "key": "client",
-            "pojo": {
-              "field": "app_id"
-            }
-          }
-        ],
+        "inputs": [],
         "api": {
           "http": {
             "method": "GET",
-            "uri": "http://localhost:8001/guest/api/{{client}}/{{user}}?format=json",
```

Comment:
Why did you remove other important test cases for placeholders? We should be adding tests with every PR, not removing them.

Reply:
placeholders weren't being used by the api, so I removed them